css/seqserv.css

```diff
@@ -9,7 +9,7 @@ body {
 }
 
 section {
-  padding-top: 50px;
```

Comment:
First, I am unable to reproduce the navbar issue you describe in #338 in Safari, Chrome, or Firefox. Second, this will increase vertical spacing throughout the page as each section of the page is encolsed in a `<section>` tag. If what you say is indeed an issue, top padding will need to be increased specifically for the first section of the page.

Reply:
You can reproduce the problem with this setting:
firefox -- about:preferences#content -- Fonts & Colors -- [Advanced...] -- [] Allow pages to choose their own fonts, instead of my selections above

This expands the navbar from one to two rows.
Because the content does not get pushed further down automatically the top navbar with two rows overlaps the content.

Possible solutions:
1. remove font size
# bad effect: font gets a little bit smaller; user can scale the content, if required
seqserv.css:2
delete: font-size: 16px !important;

2. remove width-constraints of top navbar
# bad effect: two right-most navbar items might get hidden, depending on resolution

bootstrap-3.3.6.min.css:14
disable: width: 1170px; 970px; 750px;

3. move start of content further down
# bad effect: target link, f.e. http://www.sequenceserver.com/#team, is covered by top navbar

seqserv.css:91
padding: 0 20px 20px 20px;
0 => 50px

seqserv.css:71
padding-top:10px;
10 => 50px


Is it OK to use solution 1?

Reply:
The first option will result in using Bootstrap 3's default font size of 14. Any font size below the current 16 px seems to work. Alternatively, removing the `letter-` and `word-spacing` styles also work.

I think the ideal solution will be to isolate the use of `letter-` and `word-spacing` styles to when the Source Sans Pro font (the intended font) is used, as these styles were added specifically for that font. If that's not possible, I would remove `letter-` and `word-spacing` styles altogether instead of reducing the font size to 14.